### DISCUSSION TITLE
Deformable convolution

### DISCUSSION THREAD

**Discussion by Coll1ns-cult** [open-mmlab/mmdetection#9697] [Q&A]
Hi, in the repository of deformable convolution, it is written that for pytorch implementation of the deformable convolutions, please refer to open-mmlab. I have been searching for it but haven't found it since then. If anyone else can see it, please share it with me.

**Comment by Li-Qingyun** [comment]
Hi, you can refer to https://github.com/open-mmlab/mmdetection/tree/master/configs/dcn and https://github.com/open-mmlab/mmdetection/tree/master/configs/dcnv2 for implemantations of DCN and DCNv2 papers.
The DCN in backbone has been fused as a parameter, see https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/backbones/resnet.py, the class has an initial parameter named dcn. See above config or docstring to learn the usage.
